docs/helpers.asciidoc

```diff
@@ -160,3 +160,79 @@ end
 scroll_helper.clear
 ----
 --
+
+[discrete]
```

Comment:
I'd add an anchor to this section so it'll be easier to link to it. I can imagine it'll be a popular read. :) 
```suggestion
[discrete]
[[esql-helper]]
```